bedrock/products/templates/products/vpn/includes/macros.html

```diff
@@ -108,7 +108,11 @@ <h2 class="vpn-hero-sub-heading">{{ subheading }}</h2>
     </a>
 
     <p class="availability-copy">
+    {% if switch('vpn-wave-vi') %}
+      {{ ftl('vpn-shared-available-countries-v6', fallback='vpn-shared-available-countries-v6') }}
```

Comment:
Should this fall back to v5? (same for occurrences below)
```suggestion
      {{ ftl('vpn-shared-available-countries-v6', fallback='vpn-shared-available-countries-v5') }}
```

Reply:
D'oh, paste error 🍝